docs/src/api/make_pages.py

```diff
@@ -0,0 +1,42 @@
+"""Generate the api pages and navigation.
+
+NOTE: Works best when following the Google style guide
+https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
+https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings
+"""
+
+import mkdocs_gen_files
+from pathlib import Path
+import os
+import subprocess
+
+package = os.getenv("PACKAGE")
+
+element = package.split("_", 1)[1]
+if not Path(f"workflow_{element}").is_dir():
+    try:
+        subprocess.run(
+            f"git clone https://github.com/cbroz1/workflow-{element.replace('_','-')}.git /main/delete".split(
```

Comment:
Should this be `f"git clone https://github.com/datajoint/workflow-{element.replace('_','-')}.git /main/delete"`

Reply:
good catch, that was meant to be temporary

Reply:
Yes, eventually. You can also direct it to your repo if you wanna see the changes you applied. Otherwise you won't see the changes until your PR is merged.